src/microbe_stage/organelle_unlocks/UndiscoveredOrganellesButton.tscn

```diff
@@ -0,0 +1,119 @@
+border_width_bottom = 1
+border_color = Color( 0.0666667, 1, 0.835294, 1 )
+corner_radius_top_left = 10
+corner_radius_top_right = 10
+corner_radius_bottom_right = 10
+corner_radius_bottom_left = 10
+
+[sub_resource type="StyleBoxFlat" id=2]
+bg_color = Color( 0.0666667, 1, 0.835294, 1 )
+border_width_left = 1
+border_width_top = 1
+border_width_right = 1
+border_width_bottom = 1
+border_color = Color( 0.0666667, 1, 0.835294, 1 )
+corner_radius_top_left = 10
+corner_radius_top_right = 10
+corner_radius_bottom_right = 10
+corner_radius_bottom_left = 10
+
+[sub_resource type="StyleBoxFlat" id=3]
+bg_color = Color( 0.164706, 0.215686, 0.235294, 1 )
+border_width_left = 1
+border_width_top = 1
+border_width_right = 1
+border_width_bottom = 1
+border_color = Color( 0.396078, 0.623529, 0.584314, 1 )
+corner_radius_top_left = 10
+corner_radius_top_right = 10
+corner_radius_bottom_right = 10
+corner_radius_bottom_left = 10
+
+[sub_resource type="StyleBoxFlat" id=4]
+bg_color = Color( 0.0666667, 0.168627, 0.211765, 1 )
+border_width_left = 1
+border_width_top = 1
+border_width_right = 1
+border_width_bottom = 1
+border_color = Color( 0.0666667, 1, 0.835294, 1 )
+corner_radius_top_left = 10
+corner_radius_top_right = 10
+corner_radius_bottom_right = 10
+corner_radius_bottom_left = 10
+
+[sub_resource type="ButtonGroup" id=5]
+
+[node name="UndiscoveredOrganelles" type="MarginContainer"]
+margin_right = 80.0
+margin_bottom = 103.0
+mouse_filter = 1
+custom_constants/margin_top = 5
+custom_constants/margin_bottom = 5
+script = ExtResource( 3 )
+
+[node name="VBoxContainer" type="VBoxContainer" parent="."]
+margin_top = 5.0
+margin_right = 175.0
+margin_bottom = 98.0
+mouse_filter = 2
+custom_constants/separation = 1
+
+[node name="Button" type="Button" parent="VBoxContainer"]
+margin_left = 47.0
+margin_right = 127.0
+margin_bottom = 60.0
+rect_min_size = Vector2( 80, 60 )
+focus_mode = 0
+mouse_filter = 1
+size_flags_horizontal = 4
+custom_styles/hover = SubResource( 1 )
+custom_styles/pressed = SubResource( 2 )
+custom_styles/disabled = SubResource( 3 )
+custom_styles/normal = SubResource( 4 )
+disabled = true
+action_mode = 0
+group = SubResource( 5 )
+expand_icon = true
+
+[node name="QuestionMark" type="Label" parent="VBoxContainer/Button"]
+anchor_right = 1.0
+anchor_bottom = 1.0
+custom_fonts/font = ExtResource( 2 )
+text = "QUESTION"
+align = 1
+valign = 1
+
+[node name="Count" type="Label" parent="VBoxContainer/Button"]
+anchor_left = 1.0
+anchor_right = 1.0
+margin_left = -3.0
+margin_top = 2.0
+margin_right = -3.0
+margin_bottom = 16.0
+grow_horizontal = 0
+
+[node name="Name" type="Label" parent="VBoxContainer"]
+margin_top = 61.0
+margin_right = 175.0
+margin_bottom = 76.0
+rect_min_size = Vector2( 85, 0 )
+size_flags_horizontal = 4
+custom_constants/line_spacing = -3
+custom_fonts/font = ExtResource( 1 )
+text = "UNDISCOVERED_ORGANELLES"
+align = 1
+__meta__ = {
+"_edit_use_anchors_": false,
+"_editor_description_": "PLACEHOLDER"
```

Comment:
The text seems to be a translation key so why is this label marked as untranslated?